save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -405,28 +414,23 @@ class DownloadProjectController(private val configProperties: ConfigProperties,
     private fun prepareForExecutionFromGit(project: Project,
                                            executionId: Long,
                                            propertiesRelativePath: String,
-                                           projectRootRelativePath: String): Mono<EmptyResponse> = Mono.fromCallable {
+                                           projectRootRelativePath: String,
+                                           gitUrl: String): Mono<EmptyResponse> = Mono.fromCallable {
         val testResourcesRootAbsolutePath =
                 getTestResourcesRootAbsolutePath(propertiesRelativePath, projectRootRelativePath)
         testDiscoveringService.getRootTestConfig(testResourcesRootAbsolutePath)
     }
         .log()
         .zipWhen { rootTestConfig ->
-            discoverAndSaveTestSuites(project, rootTestConfig, propertiesRelativePath)
+            discoverAndSaveTestSuites(project, rootTestConfig, propertiesRelativePath, gitUrl)
         }
         .flatMap { (rootTestConfig, testSuites) ->
             initializeTests(testSuites, rootTestConfig, executionId)
         }
     
     private fun prepareExecutionForStandard(testSuiteDtos: List<TestSuiteDto>): Mono<List<TestSuite>> {
-        return webClientBackend.makeRequest<List<TestSuiteDto>?, List<TestSuite>>(
-            BodyInserters.fromValue(
-                testSuiteDtos
-            ), "/saveTestSuites"
-        ) {
-            it.bodyToMono()
-        }
-        // fixme: should also initialize tests from standard suites
+        // FixMe: Should be properly processed in https://github.com/cqfn/save-cloud/issues/221
+        return Mono.empty()
```

Comment:
I'd suggest `Mono.just(emptyList())`, because empty mono is sort of special and will stop the processing chain if followed by `flatMap`. Here it's followed by `then`, so it still works, but still no reason to use `Mono.empty()` here

Reply:
Good point, thanks, fixed